### ISSUE TITLE
Possibility to update outdated bitflags dependency?

### ISSUE THREAD

**Issue by bluenote10** [RustAudio/rust-portaudio#192]
Currently rust-portaudio defines a dependency towards `bitflags` version 0.8.2:

https://github.com/RustAudio/rust-portaudio/blob/fa23019d4ead0079bf5fa01d6d041749d87ae387/Cargo.toml#L12

This version is over 7 years old, and the current version of `bitflags` is 2.6.0 as of this writing (see [versions](https://crates.io/crates/bitflags/versions)). When building anything with rust-portaudio the compiler already complains that this will break soon:

```
warning: the following packages contain code that will be rejected by a future version of Rust: bitflags v0.8.2
note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 2`
```

Would it be possible to bump that dependency to a newer version to stay compatible with futures versions of Rust?

Considering that rust-portaudio is a library would it make sense to not pin exact patch versions, but just major versions to let clients decide on the exact version?

**Comment by wmedrano** [comment]
`rust-portaudio` can probably upgrade to >2.0.

Also yes for latest major version.

I'll try to upgrade it today.

**Comment by wmedrano** [comment]
Closed by #193

Bitflags 2 is used starting in `portaudio` `0.8.0`

**Comment by bluenote10** [comment]
Great, I've already updated to `0.8.0` and can confirm that bitflags is up-to-date now. Thanks!

Probably safe to close the issue ticket now as well.
